src/app/components/task/DatetimeRespondTask.js

```diff
@@ -203,96 +176,103 @@ class DatetimeRespondTask extends Component {
-            name="response"
-            value={this.state.date}
-            onChange={this.handleChange.bind(this)}
-            style={styles.primaryColumn}
-            disableToolbar
-            inputVariant="outlined"
-            variant="inline"
-            format="MMMM DD, YYYY"
-            margin="normal"
-          />
-        </FlexRow>
-
-        <FlexRow style={styles.row}>
-          <div style={styles.primaryColumn}>
-            <FlexRow
-              style={{ justifyContent: 'flex-start', alignItems: 'center' }}
-              id="task__response-time"
-            >
-              <TimePicker
-                id="task__response-time-input"
+        <Box clone mb={2}>
+          <FlexRow >
+            <Box clone flex={10}>
+              <DatePicker
                 label={
                   <FormattedMessage
-                    id="datetimeRespondTask.time"
-                    defaultMessage="Pick a time"
+                    id="datetimeRespondTask.pickDate"
+                    defaultMessage="Pick a date"
                   />
                 }
-                onChange={this.handleChangeTime.bind(this)}
-                value={
-                  this.state.hour && this.state.minute ?
-                    new Date().setHours(this.state.hour, this.state.minute) : null
-                }
-                inputVariant="outlined"
                 InputProps={{
                   endAdornment: (
                     <InputAdornment>
-                      <AccessTimeIcon />
+                      <EventIcon />
                     </InputAdornment>
                   ),
                 }}
+                id="task__response-date"
+                className="task__response-input"
+                name="response"
+                value={this.state.date}
+                onChange={this.handleChange.bind(this)}
+                disableToolbar
+                inputVariant="outlined"
                 variant="inline"
-                fullWidth
+                format="MMMM DD, YYYY"
+                margin="normal"
               />
-              <Autocomplete
-                className="task__datetime-timezone"
-                style={{ marginLeft: units(2) }}
-                options={
-                  taskTimezones && taskTimezones.length ? taskTimezones : Object.values(timezones)
-                }
-                getOptionLabel={option => option.label}
-                defaultValue={this.state.timezone}
-                onChange={(event, newValue) => {
-                  this.handleChangeTimezone(newValue);
-                }}
-                renderInput={params => (
-                  <TextField
-                    {...params}
-                    variant="outlined"
+            </Box>
+          </FlexRow>
+        </Box>
+
+        <Box clone mb={2}>
+          <FlexRow >
+            <Box clone flex={10} >
+              <Box clone justifyContent='flex-start' alignItems='center'>
```

Comment:
`214:41  error  Unexpected usage of singlequote                  jsx-quotes`
`214:65  error  Unexpected usage of singlequote                  jsx-quotes`